tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = py{35,36,37}-{pytest,setup}{,-all},docs,dev,flake8
+skipsdist = True
+requires = tox-venv
+           setuptools >= 30.0.0
+
+[testenv]
+usedevelop = True
+whitelist_externals = which
+passenv = HOME LC_ALL LC_CTYPE CC CFLAGS
+description =
+    setup-all: run tests with python setup.py and all dependencies
+    setup-!all: run tests with python setup.py
+    pytest-all: run tests with pytest and all dependencies
+    pytest-!all: run tests with pytest
+    dev: run tests with numpy and matplotlib dev versions
+# declaring numpy and pytest is needed for --force-dep
+deps =
+    pytest{env:PYTEST_VERS:}
+    !dev: numpy{env:NUMPY_VERS:}
+    dev: git+https://github.com/numpy/numpy.git#egg=numpy
+    dev: git+https://github.com/matplotlib/matplotlib.git#egg=matplotlib
+extras =
+    test
+    all: all
+conda_deps=
+    Cython
+    jinja2
+    numpy{env:NUMPY_VERS:}
+    all: scipy
```

Comment:
Why is this needed? There should be a wheel, so I would have thought it would work fine without?

Reply:
You mean for scipy, or all deps here ? 
My idea was to install at least a few common packages from conda instead of pip.

Reply:
Ah ok - I think it would be good to have a comment to explain this

Reply:
Yep, done.